test/Microsoft.Azure.EventHubs.Processor.UnitTests/project.json

```diff
@@ -23,6 +23,8 @@
         "dnxcore50",
         "portable-net45+win8"
       ]
+    },
+    "net451": {
```

Comment:
Out tests don't need to be backward compatible but our client. Lets keep this net46.